src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -1693,6 +1693,7 @@ bool WindowImplX11::processEvent(XEvent& windowEvent)
                 XSetICFocus(m_inputContext);
 
             // Grab cursor
+            // TODO: ???
```

Comment:
Could you explain what these two new TODO comments mean?

Reply:
I assume this is related to:

![image](https://github.com/user-attachments/assets/2310a081-924c-4ba5-859d-30296fa8eec3)

Reply:
Yes, I don't know what the intended use of `m_cursorGrabbed` was here... Was looking for some feedback on why it was there